app/models/export.rb

```diff
@@ -40,7 +40,7 @@ def self.histogram_cm_per_pixel
 
   def self.histogram_cm_per_pixel_in_tens
     e = Export.where('cm_per_pixel != "" AND cm_per_pixel < 500')
-      .order(cm_per_pixel: 'desc')
+      .order('cm_per_pixel desc')
```

Comment:
I would prefer `cm_per_pixel: :desc` because it is rails way.
Would love to hear from you 😄 .

Reply:
@jainaman224 https://travis-ci.org/publiclab/mapknitter/builds/539536753#L1246-L1249
The test fails when I do that. Not sure why :/

Reply:
I think this is related to our MySQL adapter gem version.